src/moin/storage/middleware/indexing.py

```diff
@@ -792,6 +792,13 @@ def search_page(self, q, idx_name=LATEST_REVS, pagenum=1, pagelen=10, **kw):
                 item = Item(self, latest_doc=latest_doc, itemid=doc[ITEMID])
                 yield item.get_revision(doc[REVID], doc=doc)
 
+    def search_len(self, q, idx_name=ALL_REVS, **kw):
+        """
+        Return the number of matching revisions.
+        """
+        with self.ix[idx_name].searcher() as searcher:
+            return len(searcher.search(q, **kw))
```

Comment:
this seems somehow expensive/wasteful to run the search and only use the length of the result, but not the result itself.

Reply:
also, the method name ... - maybe rather `result_size`?